app/scripts/controllers/rif/rns/register/index.js

```diff
@@ -30,26 +30,26 @@ export default class RnsRegister extends RnsJsDelegate {
   requestRegistration (domainName, yearsToRegister) {
```

Comment:
This is maybe a too wide question but, which are the reasons to have the registration at controller level at the nifty architecture? 

Why it cannot be placed at the services layer?

Reply:
I don't know if you understood the arquitecture, but this is called by the controller, unless you want to add another useless layer this works as a service.